xwiki-rendering-api/src/main/java/org/xwiki/rendering/listener/MetaData.java

```diff
@@ -36,6 +36,16 @@
      */
     public static final MetaData EMPTY = new MetaData();
 
+    /**
+     * Defines the class to use when an element represents a MetaData block.
+     */
+    public static final String METADATA_CONTAINER_CLASS = "xwiki-metadata-container";
```

Comment:
This is a HTML thing, nothing to do here.

Reply:
I put it in there because I need the constant both in `xwiki-rendering-syntax-xhtml` and in `xwiki-rendering-syntax-annotatedxhtml` (for the renderer and the parser). Do you have any suggestion where to put else?